### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by limjermaine88**

### CODE REVIEW

**Comment by shivin9** [comment]
Did you add the user ID file?

**Comment by limjermaine88** [comment]
Hi, I have added the user ID file.

File: jermaine.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+limjermaine88@yahoo.com.sg
+limjermaine88@yahoo.com.sg
```
Comment (shivin9) [inline jermaine.txt:2]: Can you fix this?

**Review by shivin9 — CHANGES_REQUESTED** [review]

**Comment by limjermaine88** [comment]
I have fixed it

**Pull request merged**